templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -202,6 +202,17 @@ After which you should see something similar to:
 2016-07-07 15:06:59,187 INFO     127.0.0.1       POST    200 143ms (mem: 6ms/13) (md: 43ms/1) (db: 20ms/9q) (+start: 147ms) /api/v1/external/helloworld (helloworld-bot@zulip.com via ZulipHelloWorldWebhook)
 ```
 
+Using the Integrations Developer Panel (GUI):
+After doing `./tools/run-dev.py` go to http://localhost:9991/devtools/integrations/.
```

Comment:
Just make this `Run ./tools/run-dev.py and go to https://localhost.......`

Reply:
Ok :+1: